Components/badge/badge.css

```diff
@@ -0,0 +1,58 @@
+.badge{
+    position: relative;
+    display: inline-block;
+}
+
+.indicator{
+    border-radius: 50%;
+    background-color: var(--INDICATOR-GREEN);
+    position: absolute;
+    
```

Comment:
empty line and there should have space after every class name

Reply:
I didn't get your point. Could you please elaborate?